### PR TITLE
Malformed JSON when product name contains a quote character

### LINKED ISSUE #15: You need to escape string
/vendor/klaviyo/magento2-extension/view/frontend/templates/product/viewed.phtml

Does not escape the product name, produces js error if product has double quote in name.

`var item = {
            ProductID: "<?php echo $_product->getId(); ?>",
            Name: "<?php echo $_product->getName(); ?>",
            SKU: "<?php echo $_product->getSku(); ?>",
            URL: "<?php echo $_product->getProductUrl(); ?>",
            Price: <?php echo number_format($price, 2); ?>,
            FinalPrice: <?php echo number_format($final_price, 2); ?>,
            <?php if ($_product_image_url) { ?>ImageURL: "<?php echo $_product_image_url; ?>", <?php } ?>
            Categories: <?php echo $this->getProductCategoriesAsJson(); ?>
        };`

### PR DESCRIPTION
**Pull request by sippsolutions**
Fixes #15

### CODE REVIEW

File: view/frontend/templates/product/viewed.phtml [hunk 1]
```diff
@@ -1,7 +1,12 @@
 <?php
 
-/** @var \Magento\Catalog\Helper\Image $_imagehelper */
-$_imagehelper = $this->helper('Magento\Catalog\Helper\Image');
+use Magento\Catalog\Helper\Image;
+use Magento\Catalog\Model\Product;
```
Comment (remstone7) [inline view/frontend/templates/product/viewed.phtml]: do we need this as it should already be installed on the page? but if it's for being more explicit in readability, it's cool with me.

**Comment by remstone7** [comment]
looks good! just one comment

File: view/frontend/templates/product/viewed.phtml (hunk 1 above)
Comment (sippsolutions) [reply view/frontend/templates/product/viewed.phtml]: we import the classes to be able to use the short notation in the code below

**Comment by srenon** [comment]
@remstone7 ... this issue will be fixed by https://github.com/klaviyo/magento2-klaviyo/pull/14 ... 

You should close this PR and not merged into Master otherwise you going to run in a merge conflict. 

All this logic was more to https://github.com/klaviyo/magento2-klaviyo/pull/14/files#diff-18bedff8355a70ed70280a6e72eadd4dR190 and then JSON encode.

**Pull request closed without merging**